src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+:root {
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
The `.roboto-medium` class duplicates the font settings already defined for the `html` element. Consider removing this class if it's not adding any value, or ensure it's used for a specific reason that requires the redefinition of these styles.